File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int hash[100001] = {0};
        for (int i = 0; i < nums.size(); ++i)
        {
            if (hash[nums[i] + 10000] < 2)
                hash[nums[i] + 10000]++;
            else
            {
                nums.erase(nums.begin() + i);
                --i;
            }
        }
        return nums.size();
    }
};

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution(object):
    def removeDuplicates(self, nums):
        i = 1
        while i < len(nums):
            count = nums.count(nums[i - 1])
            if count > 2:
                nums.remove(nums[i - 1])
            else:
                i += 1
        return len(nums)
        """
        :type nums: List[int]
        :rtype: int
        """
        